src/v2/Apps/ViewingRoom/Components/ViewingRoomsFeaturedRail.tsx

```diff
@@ -16,19 +18,48 @@ export const ViewingRoomsFeaturedRail: React.FC<ViewingRoomsFeaturedRailProps> =
     })
     .filter(Boolean)
 
-  return featuredViewingRoomsForRail.length > 0 && (
-    <Box>
-      <Sans size="5">Featured</Sans>
+  const carouselHeight = 380
```

Comment:
No need to store this in a var since its not exported or used anywhere else, can just set it down below as a prop